llvm/test/Instrumentation/MemorySanitizer/AArch64/arm64-vadd.ll

```diff
@@ -0,0 +1,3824 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=msan -S | FileCheck %s
+;
+; Handled heuristically but incorrectly:
+; - llvm.aarch64.neon.{addp, faddp}
+;
+; Unknown instructions handled by visitInstruction:
```

Comment:
correctly or not?

Reply:
Clarified in https://github.com/llvm/llvm-project/pull/125820/commits/d707c13b869c155be412779e98a99e16ed204824

- Incorrectly handled: llvm.aarch64.neon.{addhn, raddhn}
- Suboptimally handled: llvm.aarch64.neon.{saddlp, uaddlp}